index.js

```diff
@@ -1,20 +1,15 @@
-'use strict'
-
 /**
  * @typedef {object} LoadPluginOptions
  * @property {string} [prefix]
  * @property {string | string[]} [cwd]
  * @property {boolean} [global]
  */
 
-var fs = require('fs')
-var path = require('path')
-var resolve = require('resolve-from').silent
+import fs from 'fs'
+import path from 'path'
+import {silent as resolve} from 'resolve-from'
 // type-coverage:ignore-next-line
-var readNpmConfig = require('libnpmconfig').read
-
-module.exports = loadPlugin
-loadPlugin.resolve = resolvePlugin
+import {read as readNpmConfig} from 'libnpmconfig'
```

Comment:
`libnpmconfig` relies on [`figgy-pudding`](https://github.com/npm/figgy-pudding), which is marked for deprecation. I think it’s better to look at an alternative.

Reply:
Could also be that `libnpmconfig` will swap that out for another project?